asapdiscovery-ml/asapdiscovery/ml/cli_args.py

```diff
@@ -1292,9 +1295,12 @@ def batch_size(func):
     )(func)
 
 
-def target_prop(func):
+def target_props(func):
     return click.option(
-        "--target-prop", type=str, help="Target property to train against."
+        "--target-prop",
+        type=str,
+        multiple=True,
```

Comment:
I guess this is really only like half a functionality at this point. the model will still only give one prediction for a given input, no matter how many target prop values you have. this can let you train against two values simultaneously, but it would only make sense at all if they were related (eg pIC50 and pKi), but now that I'm thinking about it idk how much sense it makes to have this functionality at all without having the model be able to make multiple predictions. do you think it's ok to leave in for now since it still works with just one prop specified, or should we get rid of it for the time being to avoid confusing people?